cdc/sink/manager.go

```diff
@@ -145,8 +146,13 @@ func (m *Manager) destroyTableSink(ctx context.Context, tableID model.TableID) e
 	return m.backendSink.Barrier(ctx, tableID)
 }
 
-func (m *Manager) getCheckpointTs() uint64 {
-	return atomic.LoadUint64(&m.checkpointTs)
+func (m *Manager) getCheckpointTs(tableID model.TableID) uint64 {
+	checkPoints, ok := m.tableCheckpointTsMap.Load(tableID)
+	if ok {
+		return checkPoints.(uint64)
+	}
+	// this function is call when something unexpected, return change level checkpoint is safe
+	return m.changeFeedCheckpointTs
```

Comment:
Could you return an error? Let's fail fast.

Reply:
Update the comments,
1. if the table flush task can not get the flush lock,  we return changeFeed level checpointTs,  the sinkNode will not update its checkpointTs if returned value is smaller than it.  
2. if error hanppened , sink manager call this func, and also returns the error